## x方向围栏
x_lim: 2.0
## y方向围栏
y_lim: 2.0
## z方向围栏
z_lim: 0.5

z_max: 2.5

delta: 0.012

height: 0.1

time_delay: 0.008

sleep_time: 100.0
